lib/krikri/enrichment.rb

```diff
@@ -0,0 +1,148 @@
+module Krikri
+  ##
+  # Mixin module for enriching a set of input_fields and setting the resulting
+  # values to a set of output fields.
+  module Enrichment
+    extend SoftwareAgent
+
+    ##
+    # The main enrichment method; passes specified input fields to
+    # #enrich_values, which must return an array of values with length equal to
```

Comment:
How does this impact enrichment stories like [#84945216](https://www.pivotaltracker.com/story/show/84945216) (deduplication) and [#83401638](https://www.pivotaltracker.com/story/show/83401638) (separate at semicolons)?